packages/app/src/runner/selector-playground/Highlight.ce.vue

```diff
@@ -0,0 +1,81 @@
+<template>
```

Comment:
ce stands for custom elements. 

Without this we cannot get the css string with `import` syntax because they're injected to <style> tag. 


It's for shadow dom and  web components.

https://vuejs.org/guide/extras/web-components.html#sfc-as-custom-element

Reply:
Interesting, did not know about this.